File: mk1/mk1/TP/Lista3/Zad2/Kwadrat.cpp
#include "Kwadrat.hpp"

Kwadrat::Kwadrat(int a, int kat) noexcept(false) {

    for (int i = 0; i < 4; i++) bok.push_back(a);
    this->bok = bok;
}

std::string Kwadrat::Nazwa() noexcept(true) {
    return "Kwadrat";
}

File: mk1/TP/Lista3/Zad1/Kwadrat.java
class Kwadrat extends Czworokat{

    Kwadrat(float a, float b, float c, float d, float alf){
        this.bok1 = a;
        this.bok2 = b;
        this.bok3 = c;
        this.bok4 = d;
        this.kat = alf;
    }

    public float Pole(){
        return bok1*bok1;
    }

    public float Obwod(){
        return bok1*4;
    }

    public String Nazwa(){
        return "Kwadrat";
    }
}